hello_world/blockchain_app/hello_module/hello_asset.js

```diff
@@ -0,0 +1,63 @@
+const {
+    BaseAsset,
+    codec,
+} = require('lisk-sdk');
+const {
+    helloCounterSchema,
+    CHAIN_STATE_HELLO_COUNTER
+} = require('./schemas');
+
+const HelloAssetID = 0;
+
+class HelloAsset extends BaseAsset {
+    name = 'helloAsset';
+    id = HelloAssetID;
+    schema = {
+        $id: '/hello/asset',
+        type: 'object',
+        required: ["helloString"],
+        properties: {
+            helloString: {
+                dataType: 'string',
+                fieldNumber: 1,
+            },
+        }
+    };
+
+    validate({asset}) {
+        if (!asset.helloString || typeof asset.helloString !== 'string' || asset.helloString.length > 64) {
+          throw new Error(
+                'Invalid "asset.hello" defined on transaction: A string value no longer than 64 characters is expected'
+            );
+        }
+    };
+
+    async apply({ asset, stateStore, reducerHandler, transaction }) {
+        const senderAddress = transaction.senderAddress;
+        const senderAccount = await stateStore.account.get(senderAddress);
+        let counter = { helloCounter: 0 };
+
+        senderAccount.hello.helloMessage = asset.helloString;
+        stateStore.account.set(senderAccount.address, senderAccount);
+
+        let counterBuffer = await stateStore.chain.get(
+            CHAIN_STATE_HELLO_COUNTER
+        );
+
+        if (counterBuffer) {
+            counter = codec.decode(
+                helloCounterSchema,
+                counterBuffer
+            );
+        }
```

Comment:
Same the buffer must be there as it was applied in genesis block, no need for this condition anymore.